worlds/undertale/Locations.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, player: int, name: str, address: typing.Optional[int], parent
     "True Lab Plot": AdvData(79063, "Hotland"),
     "Left New Home Key": AdvData(79064, "New Home"),
     "Right New Home Key": AdvData(79065, "New Home"),
+    "Starting Key": AdvData(79067, "Hub"),
```

Comment:
Since you're adding another location and we haven't completely dropped support for `data_version`, go ahead and also increment `data_version` by one in your `__init__.py` file.